system/Common.php

```diff
@@ -203,7 +204,12 @@ function command(string $command)
     /**
      * More simple way of getting config instances from Factories
      *
-     * @return object|null
+     * @template ConfigTemplate of BaseConfig
+     *
+     * @param class-string<ConfigTemplate>|string $name
+     *
+     * @return ConfigTemplate|null
```

Comment:
Probably this must be reverted.

```suggestion
     * @return object|null
```